code/datums/mutations/fire_breath.dm

```diff
@@ -71,6 +74,14 @@
 	)
 	// Try to set us to our original direction after, so we don't end up backwards.
 	living_cast_on.setDir(original_dir)
+	//If we have the empowered version it'll also knock us down, assuming we cant use our hands to remain balanced
+	//Using the throw range to check for empowering because it's easier
+	if(iscarbon(cast_on) && (self_throw_range > 1))
+		var/mob/living/carbon/lizard_projectile = cast_on
+		if(lizard_projectile.restrained())
+			lizard_projectile.Knockdown(2 SECONDS)
+			to_chat(lizard_projectile, span_warning("You can't keep your balance with your hands restrained!"))
```

Comment:
who needs their hands to keep balance
especially if they're a lizard, they literally have a tail that should be able to keep their balance for them